db/schema.rb

```diff
@@ -26,6 +26,12 @@
     t.index ["user_id"], name: "index_audits_on_user_id"
   end
 
+  create_table "barcodes", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+    t.string "prefix"
+    t.integer "counter"
+    t.index ["counter"], name: "index_barcodes_on_counter"
+  end
+
```

Comment:
Can't spot the corresponding migration for this, and can't see a matching model. Did this slip in by accident?